rust/cubesql/cubeclient/src/models/v1_cube_meta_measure.rs

```diff
@@ -23,6 +23,8 @@ pub struct V1CubeMetaMeasure {
     pub description: Option<String>,
     #[serde(rename = "type")]
     pub r#type: String,
+    #[serde(rename = "aliasMember", skip_serializing_if = "Option::is_none")]
+    pub alias_member: Option<String>,
```

Comment:
@MikeNitsenko, it's an autogenerated file; OpenSpec code gen generates it. Could you please update (openspec.yml)[https://github.com/cube-js/cube/blob/master/packages/cubejs-api-gateway/openspec.yml#L146]